astropy/utils/decorators.py

```diff
@@ -121,7 +121,7 @@ def deprecated_func(*args, **kwargs):
         # If this is an extension function, we can't call
         # functools.wraps on it, but we normally don't care.
         # This crazy way to get the type of a wrapper descriptor is
-        # straight out of the Python 3.3 inspect module docs.
```

Comment:
Maybe the mention of Python here is relevant, since `inspect` might have change or will change in future Python 3.x releases.

Reply:
OK, will add a follow-up commit once the tests pass.